README.md

```diff
@@ -76,9 +76,11 @@ Make sure you already have helm CLI installed. To install the secrets store csi
 
 ```bash
 NAMESPACE=dev
-helm install charts/secrets-store-csi-driver -n csi-secrets-store --namespace $NAMESPACE
+helm repo add kubernetes-sigs-secrets-store-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
```

Comment:
nit: can we use `secrets-store-csi-driver` instead of `kubernetes-sigs-secrets-store-csi-driver`

Reply:
Sure.